stock_orderpoint_move_link/README.rst

```diff
@@ -15,7 +15,7 @@ Usage
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/153/11.0
+   :target: https://runbot.odoo-community.org/runbot/153/12.0
```

Comment:
better try to move the README to a fragmented version: follow the template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
Thank you @lreficent will let you guys know again once it is ready, also together with purchase_move_link.

Reply:
let me know if you have any doubt while fragmenting the readme

Reply:
@lreficent I now understand how the readme works. But I have other question.

I did migrate to v12 (on my local) the **stock_orderpoint_purchase_link**, also can test it OK.
But still I don't get to test **stock_orderpoint_move_link** (even in v11). Can you guide me the test scenario? and I also can add the test script.

As you suggest, I can migrate **stock_orderpoint_purchase_link** later once I done with the first one, since it is dependency.

Reply:
You can test it enabling the reception in 2 or 3 steps and then running an orderpoint, moves should have the link to the orderpoint that generated them.

Remember: You still have fill in the fragment of the README, you now only have the template in place.

Reply:
Thank you @lreficent , I have tested and done the README. Please let me know if I need anything else.